CRM/Admin/Form/Options.php

```diff
@@ -353,6 +372,9 @@ public function buildQuickForm(): void {
     }
 
     $this->addFormRule(['CRM_Admin_Form_Options', 'formRule'], $this);
+    //need to assign custom data type and subtype to the template
+    $this->assign('customDataSubType', $this->_gid);
```

Comment:
do we? are they already assigned as 'something' I've been assigning them in the smarty include as it's closer to where it relates

Reply:
These aren't already assigned to the template I don't believe

Reply:
@seamuslee001 ok - I think maybe removed custom data type from the comment then & this is MOP

Reply:
done @eileenmcnaughton